src/components/Dialogs/RootModal.scss

```diff
@@ -13,8 +18,6 @@
   // gray-dark #43484e in hex.
   --faded-gray-dark: 67, 72, 78;
   background-color: rgba(var(--faded-gray-dark), 0.75);
```

Comment:
instead of rgba, I think this should be
`  background-color: rgb(var(--faded-gray-dark), 0.75);`

Reply:
These were updated automatically, but  I changed them based on your comments